.github/workflows/ci.yml

```diff
@@ -43,10 +43,23 @@ jobs:
       fail-fast: false
       matrix:
         NXF_VER: ${{ fromJson(needs.define_nxf_versions.outputs.matrix) }}
+        test:
+          - tests/pipeline/test_spaceranger_ffpe_v1.nf.test
+          - tests/pipeline/test_spaceranger_ffpe_v2_cytassist.nf.test
+          - tests/pipeline/test_downstream.nf.test
```

Comment:
is this to parallelize the tests ? :thinking:

Reply:
yes, spaceranger takes forever, even on a minimal test dataset.